pxtarget.json

```diff
@@ -141,6 +142,21 @@
                         "SpriteKind": "SpriteKindLegacy"
                     }
                 }
+            ],
```

Comment:
This would need to also map apis from scene.setTile to scene._setTile (and same for rest of apis) to make them call that extensions version, right?

Reply:
just updated the common-packages to move the old APIs out instead of using the shim layer, so i don't think migration is needed. sprite.place / sprite.tileHitFrom are different in the extension, but the old APIs/blocks are still present (just deprecated) so old projects should still work